script/libtest.bash

```diff
@@ -142,6 +142,11 @@ kubeless_function_delete() {
     echo_info "Deleting function "${func}" in case still present ... "
     kubeless function ls |grep -w "${func}" && kubeless function delete "${func}" >& /dev/null || true
 }
+kubeless_kafka_trigger_delete() {
+    local trigger=${1:?}; shift
+    echo_info "Deleting kafka trigger "${trigger}" in case still present ... "
+    kubeless trigger kafka list |grep -w "${trigger}" && kubeless trigger kafka delete "${trigger}" >& /dev/null || true
```

Comment:
what's the point of listing the trigger first?

Reply:
Delete only if the trigger exists.